src/base/bittorrent/session.h

```diff
@@ -145,6 +151,49 @@ namespace BitTorrent
         uint nbErrored = 0;
     };
 
+#if LIBTORRENT_VERSION_NUM >= 10100
+    struct SessionMetricIndicies
+    {
+        struct
+        {
+            int hasIncomingConnections = 0;
+            int sentPayloadBytes = 0;
+            int recvPayloadBytes = 0;
+            int sentBytes = 0;
+            int recvBytes = 0;
+            int sentIPOverheadBytes = 0;
+            int recvIPOverheadBytes = 0;
+            int sentTrackerBytes = 0;
+            int recvTrackerBytes = 0;
+            int recvRedundantBytes = 0;
+            int recvFailedBytes = 0;
+        } net;
```

Comment:
I didn't try it here but initialization of structs this way it some other files produces compiler warnings.